tox.ini

```diff
@@ -1,9 +1,6 @@
 [tox]
 envlist =
-    py{36,37,38}-django21
-    py{36,37,38}-django22
-    py{36,37,38}-django30
-    py{36,37,38}-djangomaster
+    py{36,37,38}-{dj21, dj22, dj30, dj31, djmaster}
```

Comment:
what is the point of changing django to dj?

Reply:
To type less :-p, also in this line it would have been a lot of pain(in my opinion of course ;)) to write this as:

```ini
py{36,37,38}-{django21, django22, django30, django31, djangomaster}
```

Reply:
hmmm, I agree with you for the line. Can you please revert to django full word?

Reply:
okay, so that change will have to be made throughout the file. also, if you wish i will change `py` to `python`  :-p

Reply:
The fact is I don't like dj at all

Reply:
oh, i understand you don't like electronic music :-p